src/EnergyPlus/DaylightingManager.cc

```diff
@@ -4033,37 +4038,46 @@ void GetInputIlluminanceMap(EnergyPlusData &state, bool &ErrorsFound)
 
             auto &illumMap = dl->illumMaps(MapNum);
             illumMap.Name = ipsc->cAlphaArgs(1);
-            illumMap.zoneIndex = Util::FindItemInList(ipsc->cAlphaArgs(2), state.dataHeatBal->Zone);
-
-            if (illumMap.zoneIndex == 0) {
-                ShowSevereError(state,
-                                format("{}=\"{}\", invalid {}=\"{}\".",
-                                       ipsc->cCurrentModuleObject,
-                                       ipsc->cAlphaArgs(1),
-                                       ipsc->cAlphaFieldNames(2),
-                                       ipsc->cAlphaArgs(2)));
-                ErrorsFound = true;
+            int const spaceNum = Util::FindItemInList(state.dataIPShortCut->cAlphaArgs(2), state.dataHeatBal->space);
```

Comment:
Accept a Space Name for Output:IlluminanceMap.

Reply:
The way you ordered the if-else ladder seem to imply it would be more common now to use a Space Name rather than a Zone Name. Is that so?

Reply:
> The way you ordered the if-else ladder seem to imply it would be more common now to use a Space Name rather than a Zone Name. Is that so?

Good question. I didn't pay much attention to this.
For `Daylighting:Controls` space is checked first.
For `Daylighting:ReferencePoint` zone is first.
For `Output:IlluminanceMap` space is first.
For internal gains (`Lights`, `People`, etc.) it is zone, then space, then zonelist, then spacelist.

I will change `Daylighting:Controls` and  `Output:IlluminanceMap` to check zone first, then space. That will be consistent with the order in the field name.